templates/partial/_careers-available-roles.html

```diff
@@ -1,7 +1,18 @@
 {% if vacancies %}
-<h2 class="p-heading--4 js-filter__no-results u-hide">
-  Sorry, there are no open positions matching your criteria.
-</h2>
+<div class="p-strip is-shallow u-no-padding--top js-filter__no-results u-hide">
+  <div class="row">
+    <div class="u-align--left col-6 col-start-large-2">
+      <h2>There are no open roles matching your criteria</h2>
+      <p>Why not try widening your search? You can do this by:</p>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Searching for alternative words or phrases</li>
+        <li class="p-list__item is-ticked">Filtering by different department</li>
+        <li class="p-list__item is-ticked">Trying a different spelling</li>
+      </ul>
+      <a href="/careers/all" class="p-button--positive">View all roles</a>
```

Comment:
```suggestion
      <a href="/careers/all" class="p-button">View all roles</a>
```
we already have a primary button on this page